src/mscorlib/src/System/Threading/WaitHandle.cs

```diff
@@ -448,7 +448,7 @@ public static int WaitAny(WaitHandle[] waitHandles, int millisecondsTimeout)
                                         WaitHandle toWaitOn)
         {
 #if PLATFORM_UNIX
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(SR.Arg_PlatformNotSupported); // https://github.com/dotnet/coreclr/issues/10441
```

Comment:
I'm fine with that. I figured there was some value when it was a whole technology. Eg., CAS. It also codified my mental process of figuring out why we don't support it...